src/fragments/lib/push-notifications/android/record_notifications/10_handle_notification_received.mdx

```diff
@@ -0,0 +1,100 @@
+<Callout>
+
+You do not need to add these APIs to your app if you want to use Amplify with Amazon Pinpoint. You should use these APIs if you are interested in supporting multiple push providers.
+
+</Callout>
+
+## Handle Notification Received
```

Comment:
Maybe before we dive in we can mention what do we mean by "recording" and what are the common use cases for using it?